lib/structures/benches/ord_map.rs

```diff
@@ -0,0 +1,126 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion, Throughput};
+use std::fmt;
+use std::mem;
+use structures::map::ord::Map;
```

Comment:
It could at this point. They have different interfaces and I can easily see these benchmarks diverging naturally, which'd be difficult if we unified them. Since these are two distinct types I'm a little wary of reducing duplication by generics here, though I'm not hard-no. I guess, once things are repeated 3 times that's my threshold for finding a way to reduce copies.